packages/camera/camera_avfoundation/example/ios/RunnerTests/MockCaptureDeviceController.m

```diff
@@ -0,0 +1,119 @@
+
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "MockCaptureDeviceController.h"
+
+@import camera_avfoundation;
+#if __has_include(<camera_avfoundation/camera_avfoundation-umbrella.h>)
+@import camera_avfoundation.Test;
+#endif
+@import AVFoundation;
+
+@implementation MockCaptureDeviceController
+
+- (void)setActiveFormat:(AVCaptureDeviceFormat *)format {
+  if (self.setActiveFormatStub) {
+    self.setActiveFormatStub(format);
+  }
+}
+
+- (BOOL)isFlashModeSupported:(AVCaptureFlashMode)mode {
+  return self.flashModeSupported;
+}
+
+- (void)setTorchMode:(AVCaptureTorchMode)mode {
+  if (self.setTorchModeStub) {
+    self.setTorchModeStub(mode);
+  }
+}
+
+- (BOOL)isFocusModeSupported:(AVCaptureFocusMode)mode {
+  if (self.isFocusModeSupportedStub) {
+    return self.isFocusModeSupportedStub(mode);
+  }
+  return NO;
+}
+
+- (void)setFocusMode:(AVCaptureFocusMode)mode {
+  if (self.setFocusModeStub) {
+    self.setFocusModeStub(mode);
+  }
+}
+
+- (void)setFocusPointOfInterest:(CGPoint)point {
+  if (self.setFocusPointOfInterestStub) {
+    self.setFocusPointOfInterestStub(point);
+  }
+}
+
+- (void)setExposureMode:(AVCaptureExposureMode)mode {
+  if (self.setExposureModeStub) {
+    self.setExposureModeStub(mode);
+  }
+}
+
+- (void)setExposurePointOfInterest:(CGPoint)point {
+  _exposurePointOfInterest = point;
```

Comment:
This method has the same issue of having a stub setter, but also trying to be its own stub.